### PR TITLE
New package backend

### PR DESCRIPTION
**Pull request by jamescook**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -122,7 +122,7 @@
     "jscodeshift": "^0.6.4",
     "lint-staged": "^9.3.0",
     "lodash": "^4.17.15",
-    "node-sass": "6.0.1",
+    "sass": "1.62.1",
```
Comment (jamescook) [inline package.json:125]: `node-sass` will not compile for me using nodejs 16.14.0

Comment (drewlustro) [reply package.json:125]: hmm... this monorepo is a bit delicate.

I tried running `master` locally w/ node `16.14.0` and it worked fine... not sure what to make of that. 

I'm nervous that a downstream package in the monorepo depends on `node-sass`, so let's leave this.

Comment (drewlustro) [reply package.json:125]: additionally, this change introduces a duplicate key `"sass"` – apparently already exists further down in the file. reverting...

Comment (drewlustro) [reply package.json]: lastly the build was broken 😂

**Review by drewlustro — APPROVED** [review]

**Pull request merged**